provider/identityprovider/htpasswd.go

```diff
@@ -37,6 +37,7 @@ var (
 
 	HTPasswdUsernameValidators = []validator.String{
 		stringvalidator.RegexMatches(regexp.MustCompile(`^[^/:%]*$`), "username may not contain the characters: '/:%'"),
+		stringvalidator.RegexMatches(regexp.MustCompile(`.*\S.*`), "username may not be empty string"),
```

Comment:
I think that checking the length here may make it more clear (make it simple):
```suggestion
		stringvalidator.LengthAtLeast(<minimum length>),
```

Reply:
@nirarg
there is no min length for user name. 
The regex .*\S.* is used to reject any string which only contains space, such as "", or "   ".  Maybe just needs to change the message to something like `username may not be empty/blank string`
comments?

Reply:
checking string is not empty is the same like checking length is bigger than 1, and I think it will be simpler for understanding.
Anyway, the solution suggested here looks good and its up to you if to change it.

/approve
@gdbranco please review this
Is this going to be included in v1.6.0?